src/radical/pilot/agent/scheduler/base.py

```diff
@@ -265,21 +267,25 @@ def initialize_child(self):
 
         # initialize the node list to be used by the scheduler.  A scheduler
         # instance may decide to overwrite or extend this structure.
+
+        #-----------------------------------------------------------------------
+        # Change required if the DS changes.
+
```

Comment:
All changes are required by something - but I don't think that this prominent comment will help anybody.  What's `DS`? Also, the next couple of lines are not event related to data staging...

Reply:
Removed.